tests/e2e/testcafe-tests/selector.testcafe.js

```diff
@@ -1,8 +1,8 @@
 /* global fixture, test */
 import {Selector} from 'testcafe';
 import path from 'path';
-import Eyes from '../../src/eyes';
-const configPath = path.resolve(__dirname, '../fixtures/applitools.config.js');
+import Eyes from '../../../';
+const configPath = path.resolve(__dirname, '../../fixtures/applitools.config.js');
 const eyes = new Eyes({configPath});
 
 fixture.only`Selector`.page`http://localhost:7272/helloworld.html`
```

Comment:
@amitzur Did you ever notice this? This is the only e2e test that runs. Been that way for 5 months...

Reply:
phhhh, i might have and I don't remember.